src/views/EditSidebar.vue

```diff
@@ -14,7 +13,23 @@
 		</template>
 
 		<AppSidebarTab name="Details" icon="icon-details" :order="0">
-			This is the details tab
+			<title-timepicker :event-component="eventComponent" />
+			<timezonepicker :event-component="eventComponent" />
+			<div class="section-wrapper">
+				<input v-model="location" type="text" placeholder="t('calendar', 'location')">
```

Comment:
You should use a computed property for the translation.
Additionally, if you want the value to be evaluated as javascript, you need to use :placeholder.